test/e2e-v2/cases/activemq/docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "2.1"
+
+services:
+  oap:
+    extends:
+      file: ../../script/docker-compose/base-compose.yml
+      service: oap
+    ports:
+      - "12800:12800"
+    networks:
+      - e2e
+
+  amq:
+    image: apache/activemq-classic:6.0.1
+    hostname: amq
+    ports:
+      - "61616:61616"
+      - "8161:8161"
+    expose:
+      - 1616
+    volumes:
```

Comment:
Seems JMX export of activemq is not out-of-the-box and needs some configurations, could you add this part to `setup/backend/backend-activemq-monitoring.md` doc or add a ref of this case from there.

Reply:
https://github.com/apache/skywalking/pull/12109/commits/3223307205f90fb00bb28e496eb56b0fb5b367be

updated.